packages/coinstac-ui/app/render/state/ducks/collections.js

```diff
@@ -14,7 +15,7 @@ const GET_ASSOCIATED_CONSORTIA = 'GET_ASSOCIATED_CONSORTIA';
 const REMOVE_COLLECTIONS_FROM_CONS = 'REMOVE_COLLECTIONS_FROM_CONS';
 const SET_COLLECTIONS = 'GET_ALL_COLLECTIONS';
 
-function iteratePipelineSteps(consortium, filesByGroup) {
+function iteratePipelineSteps(consortium, filesByGroup, baseDirectory) {
```

Comment:
This block was changed the most, it now replaces the paths in the first csv column with the full path using the csv dir as a base, this works for our demo files, but maybe not others.